package.json

```diff
@@ -2,9 +2,10 @@
   "name": "weather-app",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production"
```

Comment:
- [ ] [OPTIONAL] Try to also add the 'start' script from the last reviewer example. it will require you to install the webpack-dev-server as a dependency and set its config at the webpack configuration file but it's one of the most convenient and used scripts in all Javascript frameworks.